third-party/qthread/Makefile

```diff
@@ -35,8 +35,9 @@ endif
 # Have qthreads use Chapel's allocator, unless directed not to
 ifeq (, $(call isTrue, $(CHPL_QTHREAD_NO_CHPL_ALLOC)))
 CHPL_QTHREAD_CFG_OPTIONS += --with-alloc=chapel
-CFLAGS += -I$(CHPL_MAKE_HOME)/runtime/include/mem/$(CHPL_MAKE_MEM)/
-CFLAGS += -I$(CHPL_MAKE_HOME)/runtime/include/
+CFLAGS += -I$(CHPL_MAKE_HOME)/runtime/include/localeModels/$(CHPL_MAKE_LOCALE_MODEL)
```

Comment:
Is this needed because `#include "chpltypes.h"` includes `chpl-locale-model.h`? Now that we're including a relatively high level chapel header (that could easily change what headers are included) instead of the low-level one, I wonder if we should be more principled about what includes we add here. i.e. I wonder if we should use `./util/config/compileline --includes-and-defines` or something?

Reply:
Yeah, it could be time for that.  I'll give it a try.